renaissance/scripts/parsemetrics-wrk.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, 2022 IBM Corporation, RedHat and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+### Script to parse hyperfoil/wrk2 data###
+
+CURRENT_DIR="$(dirname "$(realpath "$0")")"
+source ${CURRENT_DIR}/../common.sh
+#source ${CURRENT_DIR}/../renaissance-common.sh
```

Comment:
Remove these commented lines

Reply:
I would suggest to take a look at all the commented lines in all the files and remove them if not required